### ISSUE TITLE
(Bug report) Right click menu on note does not work with subpath reverse proxy

### ISSUE THREAD

**Issue by TheAceBlock** [zadam/trilium#3452]
### Trilium Version

0.57.5

### What operating system are you using?

Other Linux

### What is your setup?

Server access only

### Operating System Version

Linux, Firefox 108 and Windows 10, Chrome 108

### Description

On a server install with reverse proxy on a subpath like shown in the [wiki](https://github.com/zadam/trilium/wiki/Server-installation#reverse-proxy-setup), if you try to right-click on any note on the note tree, no menu would show up, and the console will show an error, which shows that Trilium tried to fetch the assets file from the base URL path (`/`) rather than the configured subpath (`/trilium/`):

![Screenshot](https://user-images.githubusercontent.com/57385246/209455405-76e55d84-5817-4aa5-9a03-5c3a79f86efa.png)

Upon reading the code some bit, it seems that [this WebPack configuration](https://github.com/zadam/trilium/blob/master/webpack-desktop.config.js#L10) *might* be the one to blame, although I'm not familiar with WebPack and Node.js stuff. A similar bug can be spotted as well where after setting the password in the initial setup, the app redirects to the base URL `/` rather than the app entry, which could have something to do with [this](https://github.com/zadam/trilium/blob/master/src/routes/setup.js#L17), with a potential fix [here](https://github.com/zadam/trilium/blob/master/src/routes/login.js#L69), but I'm just guessing based on my past experiences.

### Error logs

```plaintext
Uncaught (in promise) ChunkLoadError: Loading chunk 893 failed.
(error: https://example.com/assets/v0.57.5/app-dist/893.desktop.js)
    j jsonp chunk loading:27
    e ensure chunk:6
    e ensure chunk:5
    initFancyTree note_tree.js:583
    jQuery 8
    initFancyTree note_tree.js:580
    doRender note_tree.js:235
    render basic_widget.js:79
    doRender container.js:8
    render basic_widget.js:79
    doRender container.js:8
    render basic_widget.js:79
    showWidgets app_context.js:41
    start app_context.js:32
    async* desktop.js:28
    promise callback* desktop.js:26
    <anonymous> desktop.js:142
    <anonymous> desktop.js:142
jsonp chunk loading:27:17
```

**Comment by hulmgulm** [comment]
This issue is reproducible in 0.58.0.

**Comment by hulmgulm** [comment]
This issue also exists for other resources in 0.58.0:
![image](https://user-images.githubusercontent.com/12165268/209461270-0a0d4f65-4695-498f-99cb-26d5ea809a7a.png)
![image](https://user-images.githubusercontent.com/12165268/209461280-267be43b-c41d-4d47-ad21-839f6a85fe30.png)

**Comment by zadam** [comment]
Hi, thanks for the report. Both issues fixed on the master branch - future 0.58.1. Not sure if I will make another 0.57.X release.
